code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -10,6 +10,9 @@
 		if(!(locate(/obj/structure/lattice, loc) || locate(/obj/structure/stairs, loc) || locate(/obj/structure/ladder, loc)))
 			return -1
 
+	if (species && (species.flags & IS_PLANT) && istype(DS))

```

Comment:
Change this to an isdiona() check.

Reply:
Is that even better?

https://github.com/Aurorastation/Aurora.3/blob/49eaa801a8240ed77132a4728900ef7274dceae9/code/modules/mob/mob_helpers.dm#L90-L99